lib/components/Menu/Menu.js

```diff
@@ -5,6 +5,12 @@ import MenuItem from './MenuItem';
 
 import styles from './styles/Menu.css';
 
+export const VARIANT = {
+  HORIZONTAL: 'horizontal',
+  VERTICAL: 'vertical',
+  NAV: 'nav',
+};
+
```

Comment:
These changes and the other Menu ones seem unrelated to Storybook 6

Reply:
Yea this is from some branch issue stuff. It will be removed.